torch/csrc/cuda/Module.cpp

```diff
@@ -593,10 +593,10 @@ PyObject* THCPModule_memoryStats(PyObject* _unused, PyObject* arg) {
   TORCH_CHECK(THPUtils_checkLong(arg), "invalid argument to memory_allocated");
   const auto device_index = THPUtils_unpackDeviceIndex(arg);
 
+  using c10::Stat;
```

Comment:
huh now looking at it I think it's better to put them in some namespace, maybe just `CachingAllocator`?

Reply:
Done.